examples/vision/md/nerf.md

```diff
@@ -607,206 +607,274 @@ create_gif("images/*.png", "training.gif")
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 472ms/step
 
 ```
 </div>
-![png](/img/examples/vision/nerf/nerf_11_31.png)
+    
+![png](/img/examples/vision/nerf/nerf_11_33.png)
+    
 
 
 <div class="k-default-codeblock">
 ```
+ 16/16 ━━━━━━━━━━━━━━━━━━━━ 16s 959ms/step - loss: 0.0156 - psnr: 18.1782 - val_loss: 0.0153 - val_psnr: 18.2330
 Epoch 17/20
-16/16 [==============================] - 13s 741ms/step - loss: 0.0167 - psnr: 17.9393 - val_loss: 0.0156 - val_psnr: 18.1784
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 474ms/step
 
 ```
 </div>
-![png](/img/examples/vision/nerf/nerf_11_33.png)
+    
+![png](/img/examples/vision/nerf/nerf_11_35.png)
+    
 
 
 <div class="k-default-codeblock">
 ```
+ 16/16 ━━━━━━━━━━━━━━━━━━━━ 15s 905ms/step - loss: 0.0155 - psnr: 18.1989 - val_loss: 0.0146 - val_psnr: 18.4532
 Epoch 18/20
-16/16 [==============================] - 13s 750ms/step - loss: 0.0150 - psnr: 18.3875 - val_loss: 0.0151 - val_psnr: 18.2811
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 474ms/step
 
 ```
 </div>
-![png](/img/examples/vision/nerf/nerf_11_35.png)
+    
+![png](/img/examples/vision/nerf/nerf_11_37.png)
+    
 
 
 <div class="k-default-codeblock">
 ```
+ 16/16 ━━━━━━━━━━━━━━━━━━━━ 15s 896ms/step - loss: 0.0146 - psnr: 18.4838 - val_loss: 0.0134 - val_psnr: 18.8093
 Epoch 19/20
-16/16 [==============================] - 13s 755ms/step - loss: 0.0141 - psnr: 18.6476 - val_loss: 0.0139 - val_psnr: 18.6216
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 472ms/step
 
 ```
 </div>
-![png](/img/examples/vision/nerf/nerf_11_37.png)
+    
+![png](/img/examples/vision/nerf/nerf_11_39.png)
+    
 
 
 <div class="k-default-codeblock">
 ```
+ 16/16 ━━━━━━━━━━━━━━━━━━━━ 15s 900ms/step - loss: 0.0134 - psnr: 18.8581 - val_loss: 0.0136 - val_psnr: 18.7334
 Epoch 20/20
-16/16 [==============================] - 14s 777ms/step - loss: 0.0139 - psnr: 18.7131 - val_loss: 0.0137 - val_psnr: 18.7259
+ 1/1 ━━━━━━━━━━━━━━━━━━━━ 0s 467ms/step
 
 ```
 </div>
-![png](/img/examples/vision/nerf/nerf_11_39.png)
+    
+![png](/img/examples/vision/nerf/nerf_11_41.png)
+    
 
 
 <div class="k-default-codeblock">
 ```
-100%|██████████| 20/20 [00:00<00:00, 57.59it/s]
+ 16/16 ━━━━━━━━━━━━━━━━━━━━ 16s 963ms/step - loss: 0.0129 - psnr: 19.0232 - val_loss: 0.0129 - val_psnr: 18.9893
+
+  0%|                                                                                                                                                                                                                                                                                                                                              | 0/20 [00:00<?, ?it/s]/var/tmp/ipykernel_49657/3697329163.py:127: DeprecationWarning: Starting with ImageIO v3 the behavior of this function will switch to that of iio.v3.imread. To keep the current behavior (and make this warning disappear) use `import imageio.v2 as imageio` or call `imageio.v2.imread` directly.
```

Comment:
Please fix (you can just fix the code and scrub the warning from the md file, no need to regenerate)

Reply:
Done.